### ISSUE TITLE
Document sort case insensitivity

### ISSUE THREAD

**Issue by bjmiller121** [javve/list.js#315]
We noticed that, by default, sorting is case sensitive and will put strings starting with a capital letter before strings that are not capitalized.

For example ascending sort would result in:
```
Aardvark
Deer
Moose
Zebra
anteater
bat
walrus
```

Looking through documentation, we couldn't find any options to make sort not case sensitive, however, digging through the source, we found that you can use the data attribute `data-insensitive="true"` in your sort trigger, it will make sorting case insensitive.

With this set, the above list would be sorted as:
```
Aardvark
anteater
bat
Deer
Moose
walrus
Zebra
```

Would be awesome if this could be documented so people don't have to search the source to find it.

**Comment by doublejosh** [comment]
Additionally (and how we found the feature)...
* The page load default is NOT case sensitive.
* Calling the sort method IS case sensitive.

Meaning if you don't include this data attribute and use the sort method you'll get different results on page load vs. table headers, buttons, etc.

**Comment by jakubgg** [comment]
**tl;dr**
Add the `data-insensitive="true"` to your html.

**Full explanation**
The documentation does not state it clearly, but there is additional attribute that can be added on the search buttons/table headers. 
It is `data-insensitive`. When it is set to `true` all the searches will be 'case insensitive'. The issue here is, when it is not set, the default is `false`. 

Sorting on load assumes default as `true` - and sorts 'case insensitive', but then it assumes `false` as default (if there is no `data-insensitive` attribute) and sorts differently. 

Hope that helps.

**Comment by doublejosh** [comment]
Right.  That's the bug, a basic installation will have a mis-match from page load to default sort behavior.  That's a broken default.

We also suggest documenting this feature :)

**Comment by javve** [comment]
Good catch! I'll be fix'd in the next version!
...and yes, I should put this options in the documentation :)
